storage/aptos-db-tool/src/debugger.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+#[cfg(feature = "db-debugger")]
+use aptos_db::db_debugger::{checkpoint, ledger, state_tree};
+use clap::Parser;
+
+#[derive(Parser)]
+pub enum Command {
+    #[cfg(feature = "db-debugger")]
```

Comment:
don't need this,  right? Now that you enabled the feature on the dependency from cargo.toml.

Reply:
removed